### ISSUE TITLE
Slot with choices

### ISSUE THREAD

**Issue by BoyanH** [SimGus/Chatette#17]
I am using the currently latest PyPI version (1.4.1).

The following input

```
@[entity]
	{ one / two } = one_two
	{ three / four } three_four

%[intent]
	@[entity]

```

produces the following output

```
{
  "rasa_nlu_data": {
    "common_examples": [
      {
        "entities": [
          {
            "end": 23,
            "entity": "entity",
            "start": 20,
            "value": "one_two"
          }
        ],
        "intent": "intent",
        "text": "<<CHATETTE_ENTITY>> "
      },
      {
        "entities": [
          {
            "end": 23,
            "entity": "entity",
            "start": 20,
            "value": "one_two"
          }
        ],
        "intent": "intent",
        "text": "<<CHATETTE_ENTITY>> "
      },
      {
        "entities": [
          {
            "end": 37,
            "entity": "entity",
            "start": 20,
            "value": " three  three_four"
          }
        ],
        "intent": "intent",
        "text": "<<CHATETTE_ENTITY>> "
      },
      {
        "entities": [
          {
            "end": 36,
            "entity": "entity",
            "start": 20,
            "value": " four  three_four"
          }
        ],
        "intent": "intent",
        "text": "<<CHATETTE_ENTITY>> "
      }
    ],
    "entity_synonyms": [
      {
        "synonyms": [
          " one ",
          " two "
        ],
        "value": "one_two"
      }
    ],
    "regex_features": []
  }
}
```

I noticed however, that using `{one/two} = one_two` instead of `{ one / two } = one_two` works just fine. I couldn't figure out if the latter usage is supported, I find it much more readable though.

**Comment by SimGus** [comment]
Actually, `{ one / two }` and `{one/two}` are not intended to generate the same strings because whitespaces are taken into account. What this means is that `{one/two}` could generate `one` while `{ one / two }` would generate ` one ` (note the whitespaces around the word).

That being said, consecutive whitespaces are considered as a single whitespace (meaning you will never get 2 consecutive whitespaces in the generated output).

However, your code shouldn't generate `<<CHATETTE_ENTITY>> `, that is definitely a bug. I'll check out what is up with that as soon as possible.

Thanks for your input :)

**Comment by SimGus** [comment]
This is fixed in the latest version [v1.4.2](https://github.com/SimGus/Chatette/releases/tag/v1.4.2) (which you can also find on [PyPI](https://pypi.org/project/chatette/)).
Sorry for the time it took to fix that.
